examples/tut_scan.cpp

```diff
@@ -9,7 +9,7 @@
 #include <iostream>
 #include <algorithm>
 #include <numeric>
-#include <random>
+// #include <random>
```

Comment:
If not used, any reason to not delete the commented out lines in this file (here and line 73 below)?

Reply:
I was debating that, it's a better test if the data is random but functional without it.  Could add a todo or an issue and delete these?

Reply:
They're removed now.